estate/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "estate",
```

Comment:
Odoo guideline (not sure it's an official one though): prefer simple quote for "code" strings, and double quotes for displayed ones
```suggestion
    'name': "estate",
```